chrome/content/zotero/xpcom/data/item.js

```diff
@@ -1274,7 +1274,7 @@ Zotero.Item.prototype._saveData = Zotero.Promise.coroutine(function* (env) {
 		yield Zotero.DB.queryAsync(sql, env.sqlValues);
 		
 		if (!env.options.skipNotifier) {
-			Zotero.Notifier.queue('modify', 'item', itemID, env.notifierData);
+			Zotero.Notifier.queue('modify', this._objectType, itemID, env.notifierData);
```

Comment:
ItemTreeView actually has code to handle feedItem correctly, but this wasn't set, so it was doing a lot more work than necessary. I came across this accidentally, while thinking of using notifier (didn't in the end), but may as well fix this.